src/SlackFunction.spec.ts

```diff
@@ -332,6 +370,71 @@ describe('SlackFunction module', () => {
         assertNode.rejects(shouldReject);
       });
     });
+    it('app.function.blockAction() should execute all provided callbacks', async () => {
```

Comment:
Thoughts on locating this test and the one below it within `describe('runInteractivityHandlers', ....) on ln241?